src/main/resources/data.sql

```diff
@@ -3,8 +3,12 @@ INSERT INTO `country_houses` (`id`, `country`, `name`) VALUES ('2', 'Chile', 'Ca
 INSERT INTO `country_houses` (`id`, `country`, `name`) VALUES ('3', 'Uruguay', 'Casa central de Uruguay');
 INSERT INTO `country_houses` (`id`, `country`, `name`) VALUES ('4', 'Colombia', 'Casa central de Colombia');
 
-
 INSERT INTO `accounts` (`id`, `password`, `rol`, `username`, `id_country_house_fk`) VALUES ('1', 'contra123', '1', 'user_one', '1');
 INSERT INTO `accounts` (`id`, `password`, `rol`, `username`, `id_country_house_fk`) VALUES ('2', 'contra123', '1', 'user_two', '2');
 INSERT INTO `accounts` (`id`, `password`, `rol`, `username`, `id_country_house_fk`) VALUES ('3', 'contra123', '1', 'user_three', '3');
-INSERT INTO `accounts` (`id`, `password`, `rol`, `username`, `id_country_house_fk`) VALUES ('4', 'contra123', '1', 'user_four', '4');
\ No newline at end of file
+INSERT INTO `accounts` (`id`, `password`, `rol`, `username`, `id_country_house_fk`) VALUES ('4', 'contra123', '1', 'user_four', '4');
+
+INSERT INTO `sellers` (`name`) VALUES ('Carolina Fugita');
```

Comment:
Adicionar esse INSERT depois no DBService

Reply:
DBService deixou de ser necessário após o commit [Change UUID for Long](https://github.com/gustavo-franca-meli/bootcamp-br-gp9-project/commit/06b199667640105da2cb54e0c800b660d7dd24e2)